safe/definitions/minimum_needs.py

```diff
@@ -70,9 +68,9 @@ def _initializes_minimum_needs_fields():
             field_name = '{namespace}__{field_name}'.format(**format_args)
             # It's easier to debug if we have the real value from the
             # postprocessor. That's why I put a Double. ET.
-            field_type = QVariant.Double
+            field_type = qvariant_numbers
```

Comment:
So, do you approve or not approve? 😄
If we want to make it consistent, we either make both displaced and minimum needs used either double or int.
My concern is just for the inconsistencies if the number added up to aggregation impacted.